README.md

```diff
@@ -1 +1,30 @@
-# adfhhw
\ No newline at end of file
+Task 1. Initiate GIT repository (BEF):
+1. Create a github repo with the name “csdt<YY1YY2><group><student's full
```

Comment:
the branches are wrong as well: should be merged into develop from feature/develop/task1